tools/font-subset/BUILD.gn

```diff
@@ -69,6 +70,12 @@ zip_bundle("font-subset") {
     },
   ]
 
+  if (is_mac && target_cpu == "x64") {
```

Comment:
is this not built on arm?

Reply:
umm maybe it is built on arm as well but only code signed on x64? in the old code signing script, for font-subset I only find darwin-x64. https://github.com/christopherfujino/codesign.py/blob/a0c3e4c46b936061f7b350ca33109c27e61ad17e/codesign.py#L108-L112

Reply:
to ask another way, why are we only running this block if `target_cpu == "x64"`?

Reply:
I could be wrong but I think our old code signing script only signed darwin-x64/font-subset.zip and did not sign for other zips. just font-subset in https://github.com/christopherfujino/codesign.py/blob/a0c3e4c46b936061f7b350ca33109c27e61ad17e/codesign.py#L108-L112. It didn't sign something like darwin-arm/font-subset.zip, so I figured that we probably would just sign when( is_mac && target_cpu == "x64"), which would give us a darwin-x64/font-subset.zip

Reply:
It looks like the answer is because we don't generate arm macos host artifacts yet. However, eventually we will, and if we only generate these files for the `x64` case, then this code will be wrong once we start building for macOS arm.

Reply:
umm I think Christopher mentioned earlier that it might be a good thing if the program fails when we produce new arm macos host artifacts. So that we can detect the new artifact, and adjust the path to entitlements as well, in case they change.

Reply:
> umm I think Christopher mentioned earlier that it might be a good thing if the program fails when we produce new arm macos host artifacts. So that we can detect the new artifact, and adjust the path to entitlements as well, in case they change.

which program to fail?

Reply:
codesign standalone

Yeah if we impose no limit, and later arm64 is included in entitlements.txt, then the codesign program would assume its existence is legit, and treat it as an existing binary.

If arm64 is not included in entitlements.txt, during the traversal of filesystem, codesign standalone would detect that it exists in file system, but does not exist in the configuration file (entitlements.txt), and would therefore prompt the user to update path for arm64. 

I am thinking the latter might be better?